ui/desktop/tests/integration/components/card-test.js

```diff
@@ -0,0 +1,46 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, find } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | card', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    await render(hbs`<Card />`);
+    assert.ok(find('.card'));
+  });
+
+  test('it renders with content', async function (assert) {
+    await render(hbs`
+      <Card as |card|>
+        <card.header></card.header>
```

Comment:
@randallmorey Didn't realize till now I was double checking the PR... Should I change in each test the render function and use the named blocks?